<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDlpMTFRsltDisplay</class>
 <widget class="QWidget" name="QDlpMTFRsltDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDlpMTFRsltDisplay</string>
  </property>
  <widget class="QDockWidget" name="dockWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>200</height>
    </rect>
   </property>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>1600</width>
     <height>200</height>
    </rect>
   </property>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>1600</width>
     <height>200</height>
    </rect>
   </property>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>600</y>
     <width>1600</width>
     <height>200</height>
    </rect>
   </property>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
